packages/bpk-component-boilerplate/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "bpk-component-boilerplate",
+  "version": "0.0.0",
+  "description": "Backpack boilerplate component. For reference only, not intended to be used or published.",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Skyscanner/backpack.git"
+  },
+  "author": "Backpack Design System <backpack@skyscanner.net>",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": "^15.0.0 || ^16.0.0"
+  },
+  "dependencies": {
+    "bpk-component-text": "^1.0.64",
```

Comment:
I can see myself copying this boilerplate and forgetting to remove this 😄 . I think it's better we avoid adding other bpk components in the boilerplate.